frontend/src/i18n/messages/en/index.ts

```diff
@@ -86,6 +83,7 @@ export const messages = {
         funding_details: "Funding Details",
         other: "Other",
       },
+      noDocuments: "No documents are currently available here",
```

Comment:
(nit): We are using snake case for the overwhelming majority of this file. I might make a PR to make that 100% consistent, if you agree we should use snake case for this file.

Reply:
true, it gets very confusing to have it half and half. I'll prioritize that https://github.com/HHS/simpler-grants-gov/issues/4859